### PR TITLE
created some tests

### PR DESCRIPTION
**Pull request by bezmozeg**
created a few test cases, Probably not enough but these were some that I could come up with, if you have ideas what tests I should include, please comment, idk if 6 is enough

### CODE REVIEW

File: Gliocas/templates/registration/password_reset_email.html [hunk 1]
```diff
@@ -0,0 +1,5 @@
+Dear {{user.get_username}},
+You are recieving this email because you requested a change of password on Gliocas.
+To continu with the procedure, please follow the link: {{ protocol}}://{{ domain }}{% url 'password_reset_confirm' uidb64=uid token=token %}
```
Comment (JoseAntRodriguez) [inline Gliocas/templates/registration/password_reset_email.html]: Typo on 'continu'

File: Gliocas/templates/registration/password_reset_email.html [hunk 2]
```diff
@@ -0,0 +1,5 @@
+Dear {{user.get_username}},
+You are recieving this email because you requested a change of password on Gliocas.
```
Comment (JoseAntRodriguez) [inline Gliocas/templates/registration/password_reset_email.html]: Typo on 'recieving'

**Comment by JoseAntRodriguez** [comment]
I keep getting the error 'No module named gliocas_app' when running tests.py from terminal

**Comment by bezmozeg** [comment]
you run them by running python manage.py test gliocas_app

File: Gliocas/gliocas_app/tests.py [hunk 3]
```diff
@@ -1,3 +1,281 @@
...
+        course= Course.objects.create(name='ab',subject = subject)
+        course.save()
+        user = User.objects.create(username='test')
+        user.set_password('12345')
+        user.save()
+
+        question = Question.objects.create(poster=user,course=course)
+        question.text='abcd'
+        question.title='abcd'
+        question.date = datetime.datetime.now()
```
Comment (JoseAntRodriguez) [inline Gliocas/gliocas_app/tests.py]: You should import timezone from django.utils and write in this line and others like it 'question.date = timezone.now()'

**Comment by JoseAntRodriguez** [comment]
Everything works for me and once the suggestions are implemented I'll approve it. Just one thing: now we need to create a debugging server to receive the reset password mails, but that is not something appropiate for release. Is it possible to make the website send emails to a normal account so that the users can reset their passwords using the normal way? If not, maybe we should reconsider letting users reset their passwords at all.

**Comment by bezmozeg** [comment]
implemented all your suggestions, I was able to send to my email, try if it works for your email

**Review by JoseAntRodriguez — APPROVED** [review]
The test cases don't give me any error and I could reset the password of one account. Looks ready to be pushed

**Pull request merged**